drop-boxes/register-mtb-data-dropbox/register-mtb-data-dropbox.py

```diff
@@ -350,6 +379,9 @@ def register_vcf(in_file, transaction):
     # Attach the directory to the dataset
     transaction.moveFile(in_file, data_set)
 
+    # Update sample location
+    update_sample_location_to_qbic(barcode[0])
```

Comment:
Is there a reason the barcode is not extracted from the list but the list of length 1 is kept? I would think that it would be clearer to have a `QBARCODE` instead of a `[QBARCODE]`

Reply:
valid point, I was thinking about it yesterday as well. Now, that you bring it up as well, I will refactor it directly, to make it more intuitive. Thank you @KochTobi !

Reply:
I noticed that to be the case in multiple instances here. Maybe we should refractor the other uses of variablename[0] as well?

Reply:
i just did 👍